pkg/version/version_test.go

```diff
@@ -0,0 +1,32 @@
+package version
+
+import (
+	"testing"
+
+	asrt "github.com/stretchr/testify/assert"
+)
+
+func TestGetVersionInfo(t *testing.T) {
+	assert := asrt.New(t)
+	v := GetVersionInfo()
+
+	assert.Contains(v, "cli")
```

Comment:
Aren't all these first items redundant? The assertions below will fail if the keys don't exist right?

Reply:
Probably. I wrote the first items, and then thought better of it and actually checked the values. The first tests are just leftovers. I'll remove them.